test/bigtest/tests/request-policy/request-policy-form-test.js

```diff
@@ -78,9 +78,7 @@ describe('RequestPolicyForm', () => {
         description: 'Request policy 1 desc',
         requestTypes: ['Hold', 'Page']
       });
-    });
 
-    beforeEach(function () {
```

Comment:
Oh interesting! This file has several back-to-back `beforeEach` blocks. I wonder if this is the source of the failing tests I've seen in #602.

Reply:
I suppouse it can be the reason, but not sure. Locally tests were passing, but were failing in CI until I removed `beforeEach` blocks